web/src/components/tasks/CheckElevatorRegistrationStatus.tsx

```diff
@@ -0,0 +1,146 @@
+import { Alert } from "@/components/njwds-extended/Alert";
+import { SecondaryButton } from "@/components/njwds-extended/SecondaryButton";
+import { getMergedConfig } from "@/contexts/configContext";
+import { ElevatorRegistrationSearchError } from "@/lib/types/types";
+import { ElevatorSafetyAddress } from "@businessnjgovnavigator/shared/elevatorSafety";
+import { TextField } from "@mui/material";
+import createStyles from "@mui/styles/createStyles";
+import makeStyles from "@mui/styles/makeStyles";
+import { ChangeEvent, FormEvent, ReactElement, useState } from "react";
+
+const useStyles = makeStyles(() => {
+  return createStyles({
+    zipCodeField: {
+      "& input::-webkit-clear-button, & input::-webkit-outer-spin-button, & input::-webkit-inner-spin-button":
+        {
+          display: "none",
+        },
+    },
+  });
+});
```

Comment:
[dust] - is this a new pattern that we're introducing into the code base? / If so should this be disscussed with the broader team?

Reply:
This was established in Check Status, it's actually going to be removed with the zipCode changes but I carried it over for now